iree/tools/compilation.bzl

```diff
@@ -23,7 +23,9 @@ def iree_bytecode_module(
         flags = ["-iree-mlir-to-vm-bytecode-module"],
         translate_tool = "//iree/tools:iree-translate",
         cc_namespace = None,
-        visibility = None):
+        visibility = None,
+        testonly = False,
+        **kwargs):
```

Comment:
If you specify the kwargs, you shouldn't need add these manually. You should probably also forward the kwargs to both the underlying rules (implicitly creating cc_embed_data is one of the things I hate about this macro. See https://docs.bazel.build/versions/master/skylark/bzl-style.html#macros). And it's weird that visibility here doesn't get passed to the genrule too. The only thing that need to get pulled out of kwargs are args that don't make sense to just forward to both rules. Generally you'd expect a macro to take all the standard kwargs defined in all rules.

Reply:
Fixed!